src/IC/Random.H

```diff
@@ -34,6 +38,10 @@ public:
         }
 
     };
+    static void Parse(Random& value, IO::ParmParse& pp)
```

Comment:
Compiler warns that `value` and `pp` are unused. Please comment them out like

     static void Parse(Random& /*value*/, IO::ParmParse& /*pp*/)